src/sections/dashboard/searchbar/ExpandSearchBar.js

```diff
@@ -375,28 +373,37 @@ export default function ExpandSearchBar({
                 },
               }}
             >
-              {ITEMS.map((item) => (
-                <ListItem
-                  key={item.label}
-                  sx={{
-                    flexDirection: 'column',
-                    alignItems: 'flex-start',
-                    pt: 0.5,
-                  }}
-                >
-                  {/* No has context */}
-
-                  {/* Category */}
-                  <Box sx={{ fontSize: '12px', color: '#666A72', mt: 0.5 }}>
-                    {t(item.label)}
-                  </Box>
-                  {/* Search result */}
-                  <ShowTheSearchResult
-                    searchResult={searchResult}
-                    label={item.label}
-                  />
-                </ListItem>
-              ))}
+              {Object.values(searchResult).every(
+                (_searchResult) => _searchResult.length === 0,
+              ) && (
+                <Box sx={{ fontSize: '12px', color: '#666A72', mt: 0.5 }}>
+                  {t('noResult')}
+                </Box>
+              )}
+              {!Object.values(searchResult).every(
```

Comment:
rồi sao phải tách ra chạy 2 lần duyệt vậy, ba ngôi sao k xài

Reply:
dạ ba ngôi nó báo lỗi: Expected indentation of 16 spaces but found 14.

Reply:
eslint thoi, disable đi

Reply:
em disable entire file roi

Reply:
vay sua lai luon

Reply:
Ok